README.md

```diff
@@ -4,15 +4,17 @@
 
 Node module to edit resources of Windows executables.
 
-## Docs
+## Requirements
+
+On platforms other than Windows, you will need to have [Wine](http://winehq.org)
+1.6 or later installed and in the system path.
+
+## Usage
```

Comment:
Is the plan to get rid of the API docs below and host the typedocs on a separate page?

Reply:
Eventually. Again, don't want to block on administrivia :dizzy_face: :cry: